src/test/java/io/confluent/connect/hdfs/avro/DataWriterAvroTest.java

```diff
@@ -396,7 +396,7 @@ public void testFlushPartialFile() throws Exception {
     Map<TopicPartition, Long> committedOffsets = hdfsWriter.getCommittedOffsets();
     assertTrue(committedOffsets.containsKey(TOPIC_PARTITION));
     long previousOffset = committedOffsets.get(TOPIC_PARTITION);
-    assertEquals(NUMBER_OF_RECORDS, previousOffset);
+    assertEquals(NUMBER_OF_RECORDS-1, previousOffset);
```

Comment:
nit: that tells me we don't run checkstyle on tests (?)

Reply:
@rhauch this needs spaces around `-`

Reply:
fixed